test/mirror_test.html

```diff
@@ -236,6 +262,55 @@
 console.log("expected: ", false)
 console.log("\n")
 
+//automated test
+describe('checkMirror: ', function() {
```

Comment:
Please copy in comments on what checkMirror and mirrorImages are supposed to do, because the names aren't really transparent. These are defined in main/syntaxFilters.js.